### DISCUSSION TITLE
Schedules

### DISCUSSION THREAD

**Discussion by tonyhardcode** [blakeblackshear/frigate#1067] [Ideas]
I would like to see schedules implemented into Frigate. I have an indoor cam I would like to only detect/record during certain hours. I think it would be a great feature. Thanks!

**Comment by jkyselak** [comment]
You can schedule detections and clips already by using the Frigate integration for homeassistant.
This exposes detection and clip switches that can be turned on/off with an automation or a node-red flow.

**Reply by paularmstrong** [reply]
This is the answer to schedules. Blake has mentioned in other threads that Frigate is not intended to become a full-fledged application with schedules, auth, etc. We expose all of the on/off and configurations via MQTT and/or the HTTP API, so it is recommended to use those instead.
